src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_capability_updated_notification.cc

```diff
@@ -157,6 +157,17 @@ void OnSystemCapabilityUpdatedNotification::Run() {
       [&system_capability_type,
        &initial_connection_key](const ApplicationSharedPtr app) {
         DCHECK_OR_RETURN(app, false);
+        bool is_registered = false;
+        {
+          auto accessor = app->get_registration_status_accessor();
```

Comment:
@SNiukalov can `accessor` be const ref?

Reply:
@AByzhynar 
Thanks,fixed in:https://github.com/smartdevicelink/sdl_core/pull/3348/commits/79fb507d545c2f0c2780a07f858a76aee327e60f